server/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -779,12 +778,9 @@ public void testUpdateSetInvalidGeneratedColumnOnly() {
         execute("insert into computed (ts) values (1)");
         refresh();
 
-        Asserts.assertSQLError(() -> execute(
-                        "update computed set gen_col=1745"))
-                .hasPGError(INTERNAL_ERROR)
-                .hasHTTPError(BAD_REQUEST, 4000)
-                .hasMessageContaining(
-                        "Given value 1745 for generated column gen_col does not match calculation extract(YEAR FROM ts) = 1970");
+        // TODO: restore old behavior, throw an exception if no row was updated and UPDATE hit > 0 rows.
```

Comment:
This is the behaviour which is covered by test and I was not sure we should change. If its fine I will remove TODO

Reply:
I think it's fine to change but only if we treat this as a feature, not a bugfix. 
It risks breaking client applications who currently depend on it raising an error.

And we should also make a note in https://github.com/crate/crate/issues/12218 to consider update as well

Reply:
I added a note into 5.5.0 breaking change section and removed it from 5.4.3 (so won't be backporting this to 5.4).

>And we should also make a note in https://github.com/crate/crate/issues/12218 to consider update as well

Thanks for the reminder, I will add a comment there once the phrasing of the change is approved here.